bfex/components/scrapper/scrapper.py

```diff
@@ -0,0 +1,22 @@
+from bs4 import BeautifulSoup
+import urllib3
+
+urllib3.disable_warnings()
+http = urllib3.PoolManager()
+
+
+# this is where we would do json parsing to get the names?!?!?!
+
+# concatenate names into url? here ?!?!!?
+
+url = "https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/william-allison"
+response = http.request("GET", url)
+soup = BeautifulSoup(response.data, "html.parser")
+links = soup.find_all("a")
+for link in links:
+    if 'ORCID ID' in link.text:
```

Comment:
I would not do it this way. I've seen examples where professors had tagged the link differently. ie 'OrcID' 'OrcID ID'

I would look at the URL of the link, as it should point to the Orc website, and you can extract that id from there.